src/v2/Apps/Auction/Routes/__tests__/ConfirmBid.jest.tsx

```diff
@@ -182,17 +182,14 @@ describe("Routes/ConfirmBid", () => {
         confirmBidBidderPositionQueryWithWinning
       )
 
-      setTimeout(() => {
-        expect(mockBidderPositionQuery).toHaveBeenCalledTimes(1)
-        expect(mockBidderPositionQuery.mock.calls[0][1]).toEqual({
-          bidderPositionID: "pending-bidder-position-id-from-polling",
-        })
-
-        expect(window.location.assign).toHaveBeenCalledWith(
-          `/artwork/${ConfirmBidQueryResponseFixture.artwork?.slug}`
-        )
-        done()
-      }, 1001)
+      await new Promise(resolve => setTimeout(resolve, 1001))
```

Comment:
I don't like using timers in this case, I tried to use jest timers fn but wasn't working and was consuming so much time. Refactored this part because by default jest doesn't allow anymore done unless you disable the linting rule